mailbox/api/src/main/java/org/apache/james/mailbox/model/Attachment.java

```diff
@@ -107,6 +107,10 @@ public InputStream getStream() throws IOException {
         return new ByteArrayInputStream(bytes);
     }
 
+    public byte[] getBytes() {
+        return Arrays.copyOf(bytes, bytes.length);
```

Comment:
I agree with @chibenwa the risk is low.
If you want to keep this copy, do that in the constructor; so you will not do a copy on each _getBytes_ call.